### DISCUSSION TITLE
Ledger Live on Chrome OS - any update on future plans?

### DISCUSSION THREAD

**Discussion by tester2011** [LedgerHQ/ledger-live-desktop#4381] [General]
Hi,

Is there any plan to support Ledger Live on Chrome OS?

I can install and run the app from Goolge Play, everything works fine apart from the major 'roadblock' of being unable to 'see'  the Ledger Nano S connected to the USB port.

Are ther any plans to add support from ChromeOS any time soon?

Cheers,

Mike.

**Comment by aj0992** [comment]
I am also keen to understand this as unable to connect to MM extension. I understand running the Chromebook on Linux is an option but struggling to get my head around it
